### PR TITLE
Add: Every XML for Mustachesubstitution

### LINKED ISSUE #28: Header and Footer are not parsed
If you set Mustache in the Header or Footer they will not be parsed.

In the docx-zip are seperate xml-files for these.

### PR DESCRIPTION
**Pull request by maffmeier**
Resolves: #28

### CODE REVIEW

File: src/WrkLst/DocxMustache/DocxMustache.php [hunk 1]
```diff
@@ -485,7 +499,7 @@ public function SaveAsPdf()
         } else {
             $path_parts = pathinfo($this->StoragePath($this->local_path.$this->template_file_name));
 
-            return $this->StoragePath($this->local_path.$path_parts['filename'].'pdf');
+            return $this->StoragePath($this->local_path.$path_parts['filename'].'.pdf');
```
Comment (tobiasvielmetter) [inline src/WrkLst/DocxMustache/DocxMustache.php]: this seems like a breaking change. could you explain? no breaking changes please. or tackle them in a separate issue if you feel this is something that needs to be changed.

Comment (maffmeier) [reply src/WrkLst/DocxMustache/DocxMustache.php]: sorry that one slipped through. In the path the dot between filename and extension was missing

Comment (maffmeier) [reply src/WrkLst/DocxMustache/DocxMustache.php]: I can remove it from this PR if you want me to.

Comment (tobiasvielmetter) [reply src/WrkLst/DocxMustache/DocxMustache.php]: the class is used in production, so any PR should have no breaking changes, or they need to be documented. Please go through your PR and make sure it contains no refactoring leading to breaking changes, as that will lead to problems in the production environments and have to be saved for a major version.

**Pull request merged**